src/lib/menu/menu-trigger.ts

```diff
@@ -218,6 +220,7 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
     overlayState.hasBackdrop = true;
     overlayState.backdropClass = 'cdk-overlay-transparent-backdrop';
     overlayState.direction = this.dir;
+    overlayState.scrollStrategy = new RepositionScrollStrategy(this._scrollDispatcher);
```

Comment:
For a follow-up PR: rather than having menu, select, etc. all directly use `RepositionScrollStrategy`, we should have a provider that can figure the behavior more globally.